code-security/admin_guide/get-started/connect-your-repositories/add-private-registries.adoc

```diff
@@ -0,0 +1,79 @@
+:topic_type: task
+
+[.task]
+== Add Private Registries to Prisma Cloud Code Security
+
+Private Registries integration with Prisma Cloud console enables you to find an accurate dependency tree of the packages in your private registries. The accuracy into your registries, ensures you have better fix suggestions for potentially vulnerable CVE.
+On Prisma Cloud scans are run on  both Private registries and public registries giving you a more concise scan results and fix suggestions.
+To remediate vulnerabilities found on Private registries see results and fix suggestions on *Projects*. Alternatively, you can make an informed decision by seeing the scan results on *Supply Chain*, where the graph displays the dependency tree of the latest private integration.
+
+[.procedure]
+
+. Verify prerequisites.
++
+For Private Registries integration on Prisma Cloud Code Security, you need permissions and must have Prisma Cloud IP addresses and hostname on an allow list.
++
+* Access to the Prisma Cloud console.
++
+Ensure  your private registries have access to the Prisma Cloud IP addresses and hostname. You need access to 44.237.183.44 and 35.82.113.182. For more information see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.html#id7cb1c15c-a2fa-4072-%20b074-063158eeec08[enable access to the Prisma Cloud Console].
++
+* Permissions required to see scan results after integration.
++
+On the console you can see the scan results of Private registries at two instances.
++
+* *Projects*: Using the read-only permissions to private registries, you can see scan results for  Software Composition Analysis (SCA) on the console.
+* *Supply Chain*: Using the admin permission to  private registry, you can see the dependency tree on Supply Chain and make an informed decision.
+
+. Integrate Private Registries with Prisma Cloud.
+
+.. Select *Settings > Repositories > Add Repository > Private Registries*.
+
+.. Select *Artifactory*.
++
+image::private-reg.png[width=800]
+
+.. Add *Public Artifactory URL*.
++
+image::private-reg-1.png[width=600]
+
+.. Add *Username* and *Password*.
++
+image::private-reg-2.png[width=600]
++
+NOTE: The permissions associated with the username determine the scan results for private registries.
+
+.. Select *Next*.
+
+. Add Package Manager to configure registry scans on the console.
+
+.. Select a *Package Manager* to configure as a default integration.
++
+image::private-reg-3.png[width=600]
+
+.. Select *Registries* that package managers use.
++
+image::private-reg-4.png[width=600]
++
+NOTE: NPM requires you to define your repository as Private. An undefined NPM is by default a public repository.
++
+Optionally, you can select *Mirror Registry* to mirror the default integration.
+
+.. Select *Add Package*.
```

Comment:
```suggestion
.. Select *Select Package Manager Settings*.
```

Reply:
@tsmithv11 has the UI changed for this step? If yes can I see it on app-stage?